app/authenticate/templates/authenticate/authenticate.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% load static %}
+
+{% block content %}
+<form method="post">
+    {% csrf_token %}
+
+    <h1>register <br></h1>
+
+    <h5>Please sign up</h5> {% for field in form %}
```

Comment:
Could we format these `{%for ..%}` blocks a little differently so it's easier to see the nesting of the blocks? Perhaps just more this first block to below the `<h5>`, then break the two `{% endfor %}` tags on line 17 on to two lines.